service-runner.js

```diff
@@ -252,6 +251,8 @@ ServiceRunner.prototype._runWorker = function() {
         process.chdir(cwd);
     });
 
+    self._metrics = makeStatsD(self.config.metrics, self._logger);
```

Comment:
Need to add the check for `self.config.metrics.name` (from L#76), and set it to the value of L#56